Makefile

```diff
@@ -21,11 +21,12 @@ XGOIMAGE = statusteam/xgo:$(XGOVERSION)
 XGOIMAGEIOSSIM = statusteam/xgo-ios-simulator:$(XGOVERSION)
 
 networkid ?= StatusChain
+e2e_extraflags =
```

Comment:
any reason not to use it for unit tests?

Reply:
Not at all. Are you suggesting we also add a `test-unit-race` target?